bedrock/thunderbird/templates/thunderbird/organizations.html

```diff
@@ -0,0 +1,44 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "thunderbird/base-resp.html" %}
+
+{% block page_title %}{{ _('Thunderbird Extended Release Support for Your Organization, Business, Enterprise - Overview') }}{% endblock %}
+
+{% block body_id %}thunderbird-organizations{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'thunderbird-organizations' %}
+{% endblock %}
+
+{% block content %}
+  <header id="main-feature" class="stacked">
+    <h1>{{ _('Mozilla Thunderbird <span class="large">Extended Support Release</span>') }}</h1>
+    <h4>{{ _('A community-lead project that allows organizations to benefit from the speed, flexibility and security of Thunderbird while getting the support they need.') }}</h4>
+  </header>
+
+  <main id="main-content">
```

Comment:
hmmm, it seems like there is a bit of an inconsistency in our use of `role="main"` in the repo. A lot of pages have it, while there are also those that do not. For the sake of greater consistency, I would vote for adding the role attribute here. See the browser support section here for more: http://html5doctor.com/the-main-element/